src/components/AnnotationSelector.vue

```diff
@@ -225,7 +233,8 @@ export default {
       return newVal
         // exclude annotations from selected categories
         .filter(a => !sc.has(a.split(SEPARATOR, 2)[0]))
-        .map(a => this.annotationById[a]);
+        .map(a => this.annotationById[a])
+        .filter(a => a);
```

Comment:
I would add a warning here that the term is not selectable.

Reply:
commit 7e6bdbe4f8efeab6d293e552da061f69576372d4